List.lua

```diff
@@ -155,49 +155,54 @@ function lls.List:Print()
 
     -- Append last line
     if ZoUTF8StringLength(summary) > ZoUTF8StringLength(self.prefix) then
-        table.insert(lines, summary)
+        tableInsert(lines, summary)
     end
 
     -- Print to chat
-    for _, line in ipairs(lines) do
+    for i, line in ipairs(lines) do
         self.chat:Print(self.prefix .. line .. self.suffix)
+        lines[i] = nil -- to allow collect garbage
```

Comment:
I'm pretty sure that this is unnecessary.  The lines are just strings, and they will be automatically removed when `lines` goes out of scope.

Reply:
indeed, but like this, it goes out of scope directly without an additional effort